examples/authentication/client/src/main.rs

```diff
@@ -77,127 +93,20 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             .unwrap();
     });
 
-    let request_url = get_authentication_request_url(opt);
     info!("Opening Auth request in Browser");
     info!("URL: {}", &request_url);
     // Open the URL in the system-configured default browser.
     open::that(request_url)?;
 
     // Wait for result sent by redirect handler.
-    let result = runner.block_on(result_receiver.recv());
+    let result = result_receiver.recv().await;
     result_receiver.close();
 
-    // Notify server to shutdown.
+    // Notify local HTTP server to shutdown.
     shutdown_sender.send(()).unwrap();
 
-    // Wait until graceful shutdown is completed..
-    runner.block_on(task)?;
-
+    // Wait until graceful shutdown is completed, ignoring shutdown errors.
+    task.await.ok();
```

Comment:
If the intent of `ok()` is to ignore the return value, I think `let _ = task.await;` would be more explicit. Or you may as well match on the `Err` case and log it :)